### PR TITLE
fix/git-notes-AN

### PR DESCRIPTION
**Pull request by AishMckinsey**
Question 1, 3, 4, 7, 13

### CODE REVIEW

File: git-notes.md [hunk 1]
```diff
@@ -1 +1,36 @@
 <!-- Write your notes here -->
+
+1. Verison Control : manages changes to documents, computer program
```
Comment (jigishak) [inline git-notes.md]: VCS full form

File: git-notes.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+        way to track the changes in code made, who made the changes and what all changes were made  
+
+3. Branches are created from the main stable branch when we are adding a new   feature or making changes to the exisiting features
+    These changes can break the code, therefore to keep the main branches safe from our possible unstable code and for easy reviewing by other contributors upon merging.
+
+
+4.  fork is creating a copy of the exisiting code
```
Comment (aishwarya18sr) [inline git-notes.md]: Code and be changed to repository (in the first line)

File: git-notes.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+    independent of the original repository and can persist even after the original repo is deleted
+    branch is set of commits within a repo - they cannot exist without the repo
+
+7. git pull - to get the changes made in the original repo 
+              to keep our branch up to date 
+              merges the changes made in the original with our own repo
+
+    git fetch - does not merge the changes
+    git fetch --dry-run is to see the changes made in the remote branch before merge
```
Comment (aishwarya18sr) [inline git-notes.md]: git fetch --dry -run gives the summary of commits / deletions / updations

**Review by jigishak — CHANGES_REQUESTED** [review]

File: git-notes.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+13.  cd - change directory 
+        cd folder_name : changes to folder_name 
+        cd .. or cd -: goes back to the parent directory
+
+    mkdir - make directory
+
+    ls - list file
+    ls -a : for hidden files
```
Comment (aishwarya18sr) [inline git-notes.md]: ls-a : all the files including hidden files

**Review by aishwarya18sr — CHANGES_REQUESTED** [review]

**Review by aishwarya18sr — APPROVED** [review]

**Review by jigishak — APPROVED** [review]

**Pull request merged**